packages/sanity/src/core/form/inputs/InvalidValueInput/InvalidValueInput.tsx

```diff
@@ -59,7 +60,12 @@ export const InvalidValueInput = forwardRef(
       [value, actualType, validTypes]
     )
 
-    if (typeof value === 'object' && value !== null && !('_type' in value)) {
+    if (
+      isPlainObject(value) &&
+      typeof value === 'object' &&
+      value !== null &&
+      !('_type' in value)
```

Comment:
Not that it probably matters all that much _in practice_, but we're kinda doing the same job twice here to make TypeScript happy. I think it's perfectly fine to use `!('_type' in (value as object))` here